spec/API_specification/signatures/_types.py

```diff
@@ -37,4 +38,4 @@ class iinfo_object:
 
 __all__ = ['Any', 'List', 'Literal', 'NestedSequence', 'Optional',
 'PyCapsule', 'SupportsBufferProtocol', 'SupportsDLPack', 'Tuple', 'Union',
-'array', 'device', 'dtype', 'ellipsis', 'finfo_object', 'iinfo_object']
\ No newline at end of file
+'array', 'device', 'dtype', 'ellipsis', 'finfo_object', 'iinfo_object', 'Enum']
```

Comment:
Can you make sure for the Python files at least that they always end in a blank line.

Reply:
I wonder if we should add an EditorConfig file to the spec repo to avoid issues such as this.